packages/compiler-cli/src/ngtsc/diagnostics/src/error_code.ts

```diff
@@ -159,6 +159,16 @@ export enum ErrorCode {
    * An injectable already has a `ɵprov` property.
    */
   INJECTABLE_DUPLICATE_PROV = 9001,
+
+  // 1XXXX error codes are reserved for diagnostics with category
```

Comment:
Should this be `10XXX`?

```suggestion
  // 10XXX error codes are reserved for diagnostics with category
```

Reply:
good catch, thanks for the fix Pete!